packages/tokens/src/tokens/color/component-dark.json

```diff
@@ -1,7 +1,7 @@
 {
-  "button-primary-background-on-dark": {
-    "name": "button-primary-background-on-dark",
-    "value": "{semantic-test-on-dark}",
+  "vads-segmented-control-color-surface-selected-on-dark": {
+    "name": "vads-segmented-control-color-surface-selected-on-dark",
```

Comment:
Maybe answered my own question for color--seeing literally all VADS tokens seem to include it, presumably because html/css just drops in the token name vs ours being referenced `Colors.vadsColor...`.

Reply:
Yeah, all VADS semantic color tokens include "vads-color" at the beginning of their name. All component tokens include "vads-component-color-" at the beginning. This is the [article](https://medium.com/eightshapes-llc/naming-tokens-in-design-systems-9e86c7444676) that they've based some of their naming on.